HealthcareSystem/PatientWebApp/Validators/PatientValidator.cs

```diff
@@ -15,66 +15,93 @@ public class PatientValidator
             {
                 throw new ValidationException("Name is required field.");
             }
-            if (patientDTO.Surname == null || patientDTO.Surname == "")
+            if (CheckIfFieldEmpty(patientDTO.Surname))
             {
                 throw new ValidationException("Surname is required field.");
             }
-            if (patientDTO.Jmbg == null || patientDTO.Jmbg == "")
+            if (CheckIfFieldEmpty(patientDTO.Jmbg))
             {
                 throw new ValidationException("Unique identification number is required field.");
             }
-            if (patientDTO.Phone == null || patientDTO.Phone == "")
+            if (CheckIfJmbgInvalid(patientDTO.Jmbg))
+            {
+                throw new ValidationException("Unique identification number must have 13 digits.");
+            }
+            if (CheckIfFieldEmpty(patientDTO.Phone))
             {
                 throw new ValidationException("Phone number is required field.");
             }
-            if (patientDTO.CountryName == null || patientDTO.CountryName == "" ||
-                patientDTO.CityName == null || patientDTO.CityName == "" ||
-                patientDTO.HomeAddress == null || patientDTO.HomeAddress == "")
+            if (CheckIfFieldEmpty(patientDTO.CountryName) || CheckIfFieldEmpty(patientDTO.CityName) || CheckIfFieldEmpty(patientDTO.HomeAddress))
             {
                 throw new ValidationException("Country, city and home address are required fields.");
             }
-            if (patientDTO.HasInsurance && (patientDTO.Lbo == null || patientDTO.Lbo == ""))
+            if (patientDTO.HasInsurance && CheckIfFieldEmpty(patientDTO.Lbo))
             {
                 throw new ValidationException("Personal number of the insured is required field if You have insurance.");
             }
-            if(patientDTO.Email == null || patientDTO.Email == "")
+            if(CheckIfFieldEmpty(patientDTO.Email))
             {
                 throw new ValidationException("Email is required field.");
             }
-            if (patientDTO.Password == null || patientDTO.Password == "")
+            if (CheckIfEmailInvalid(patientDTO.Email))
+            {
+                throw new ValidationException("Your email isn't valid. Please check it and try again.");
+            }
+            if (CheckIfFieldEmpty(patientDTO.Password))
             {
                 throw new ValidationException("Password is required field.");
             }
+            if (CheckIfPasswordInvalid(patientDTO.Password))
+            {
+                throw new ValidationException("Your password isn't valid. It must be between 8 and 20 characters.");
+            }
+        }
+
+        private bool CheckIfObjectEmpty(PatientDTO patientDTO)
+        {
+            if (patientDTO == null) return true;
+            return false;
+        }
+
+        private bool CheckIfFieldEmpty(string field)
```

Comment:
Why not just use String.IsNullOrEmpty here? I'm not going to request changes over that but it might be nicer.